code/game/gamemodes/blob/blobs/heatsink.dm

```diff
@@ -0,0 +1,24 @@
+/obj/effect/blob/heatsink
+	name = "heatsink blob"
+	icon_state = "blank_blob"
+	desc = "A fleshy mass capable of asborbing laser fire."
+	health = 5
+	maxhealth = 5
+	atmosblock = 1
```

Comment:
You should override the fire_act() so it doesn't burn if you put it on top of a fire.